src/test/javascript/spec/component/exercises/shared/exercise-scores.component.spec.ts

```diff
@@ -108,12 +100,12 @@ describe('Exercise Scores Component', () => {
                 MockPipe(ArtemisDatePipe),
             ],
             providers: [
-                { provide: TranslateService, useClass: MockTranslateService },
                 { provide: ActivatedRoute, useValue: route },
-                { provide: AlertService, useClass: MockAlertService },
-                { provide: Router, useValue: router },
                 { provide: LocalStorageService, useClass: MockSyncStorage },
                 { provide: SessionStorageService, useClass: MockSyncStorage },
+                { provide: ExerciseService, useClass: MockExerciseService },
+                { provide: AccountService, useClass: MockAccountService },
```

Comment:
```suggestion
```
The `ArtemisTestModule` already provides the `AccountService`.

Reply:
Thanks for feedback, changed in 06c22c4.